setup.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/sh
+
+apt install python3-pip nginx net-tools libssl-dev -y
+
+python3 -m pip install -r requirements.txt
+
+# tls-perf
+git clone https://github.com/tempesta-tech/tls-perf.git /tmp/tls-perf
+cd /tmp/tls-perf
+make
+ln -s /tmp/tls-perf/tls-perf /bin/tls-perf
+
+# wrk
+git clone https://github.com/wg/wrk.git /tmp/wrk
+cd /tmp/wrk
+make
```

Comment:
Does make installs `wrk` by standard path?

Reply:
Yes it is